mrmustard/lab_dev/states/base.py

```diff
@@ -330,6 +331,12 @@ def phase_space(self, s: float) -> tuple:
         new_state = self >> BtoPS(self.modes, s=s)
         return bargmann_Abc_to_phasespace_cov_means(*new_state.bargmann_triple(batched=True))
 
+    def quadrature_distribution(self, quad: Vector, phi: float = 0.0) -> tuple | ComplexTensor:
+        r"""
+        The (discretized) quadrature distribution of the circuit component.
```

Comment:
```suggestion
        The (discretized) quadrature distribution of the State.
```
Unless we want to move this to CircuitComponent

Reply:
Also, can Args be included 🙇